scripts/fix_db/current.py

```diff
@@ -33,7 +35,10 @@
                     f"[Job {job.id:4d}] project_id=None, use dummy data"
                 )
                 project_dump = dict(
-                    id=-1, name="__UNDEFINED__", read_only=True
+                    id=-1,
```

Comment:
Reminder: this code branch is not currently tested within `migrations.yml`.
Let's modify the test DB.

Reply:
Opened its own issue #1113